modules/api/src/main/java/org/eclipse/xpanse/api/controllers/ServiceTemplateApi.java

```diff
@@ -100,12 +100,17 @@ public ServiceTemplateChangeInfo register(@Valid @RequestBody Ocl ocl) {
     @ResponseStatus(HttpStatus.OK)
     @Transactional
     @AuditApiRequest(methodName = "getCspFromRequestUri")
-    public ServiceTemplateDetailVo update(
+    public ServiceTemplateChangeInfo update(
             @Parameter(name = "id", description = "id of service template") @PathVariable("id")
-            String id, @Valid @RequestBody Ocl ocl) {
-        ServiceTemplateEntity templateEntity =
-                serviceTemplateManage.updateServiceTemplate(UUID.fromString(id), ocl);
-        return convertToServiceTemplateDetailVo(templateEntity);
+            String id,
+            @Parameter(name = "isRemoveServiceTemplateUntilApproved", description =
+                    "If remove the service template from catalog until the updated one is approved")
```

Comment:
the description is good. just one small extension to make it more explicit.

If true, the old service template is also removed from catalog until the updated one is reviewed and approved